src/app/blog/index.html

```diff
@@ -4,6 +4,39 @@
   windowTitle="SKY UX Blog">
 
 	<article class="blog-post">
+		<header>
+			<h2 class="sky-section-heading">Design update — June 2019</h2>
+		</header>
+		<section>
+			<h3 class="sky-subsection-heading">Recently completed</h3>
+				<h4 class="sky-emphasized"><a routerLink="/components/phone-field">Phone input field</a></h4>
+					<p>We ported the phone input field from SKY UX 1.</p>
+        <h4 class="sky-emphasized">Design proposals</h4>
+					<ul>
+						<li><a href="https://github.com/blackbaud/skyux-forms/issues/51">Input field masking</a></li>
+					</ul>
+		</section>
+		<section>
+			<h3 class="sky-subsection-heading">In the works</h3>
+        <h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux-list-builder/issues/22">List view switcher</a></h4>
+					<p>We are porting the list view switcher from SKY UX 1.</p>
+        <h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux2/issues/2331">Data entry grid</a></h4>
+          <p>We are working on SKY UX-themed stylesheets and configuration files for use with the ag-grid library.</p>
+        <h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux2/issues/724">Split view</a></h4>
+          <p>We are building a responsive container to hold a list of items and a workspace for users to complete tasks for the selected item.</p>
+        <h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux-forms/issues/23">Toggle switch</a></h4>
+          <p>We are building a toggle switch input for when users need to change a status outside the context of a larger form.</p>
```

Comment:
We are building a toggle switch input for scenarios when users need to change a status outside the context of a larger form.